src/components/Tooltip/Tooltip.tsx

```diff
@@ -2,59 +2,37 @@ import Tippy from '@tippyjs/react';
 import React from 'react';
 
 import { tooltipStyle } from './Tooltip.style';
+import { TooltipProps } from './Tooltip.types';
 
-export type TooltipSize = 'large' | 'medium' | 'small';
+const Tooltip = React.forwardRef<HTMLDivElement, TooltipProps>(
+  (
+    {
+      children,
+      content,
+      placement = 'right',
+      isInverted = false,
+      isInteractive = false,
+      delayIn = 500,
+      delayOut = 500,
+    },
+    ref
+  ) => {
+    return (
+      <Tippy
```

Comment:
test ids and rest?

Reply:
test-ids: `<Tippy/>` component doesn't accept data-testid as a prop ([more info here](https://github.com/atomiks/tippyjs-react/issues/89)). In order to have test-id I guess the user would have to use the interactive tooltip and add test-id to the custom content.

rest: no need for rest prop, all tooltip props are extracted and used specifically inside the component.